cognite/neat/_data_model/models/dms/_validation.py

```diff
@@ -115,6 +111,48 @@ def run(self) -> list[ConsistencyError]:
         ]
 
 
+class VersionSpaceInconsistency(DataModelValidator):
+    """This validator checks for inconsistencies in versioning and space among views and data model"""
+
+    code = "NEAT-DMS-003"
+
+    def __init__(
+        self,
+        data_model_reference: DataModelReference,
+        view_references: list[ViewReference],
+    ) -> None:
+        self.data_model_reference = data_model_reference
+        self.view_references = view_references
+
+    def run(self) -> list[Recommendation]:
+        recommendations = []
+
+        for view_ref in self.view_references:
+            issues = []
+
+            if view_ref.version != self.data_model_reference.version:
```

Comment:
Suggest we make an exception if `view_ref.space.startswith("cdf")` as Cognite force people to include core views for consistent GraphQL, and we do not want to give peolpe a warning about that.

```suggestion
            if view_ref.version != self.data_model_reference.version and not view_ref.space.startswith("cdf"):
```

Reply:
updated